src/status_im/native_module/core.cljs

```diff
@@ -40,8 +40,11 @@
 (defn login [address password callback]
   (module-interface/-login rns-module address password callback))
 
-(defn approve-sign-requests [hashes password callback]
-  (module-interface/-approve-sign-requests rns-module hashes password callback))
+(defn approve-sign-request [id password callback]
+  (module-interface/-approve-sign-request rns-module id password callback))
+
+(defn approve-sign-request-with-args [id password gas gas-price callback]
```

Comment:
why do we need two methods? maybe we could just set gas and gas-price `nil` and do not use `approve-sign-request` ?

Reply:
It's a constraint on go side and I prefer to keep 1:1 between go endpoints and native module functions.